### ISSUE TITLE
the format of TXT format,

### ISSUE THREAD

**Issue by kkwong95** [wangzhe0623/Object-Detection-Evaluation-Tool#1]
@wangzhe0623 can you provide more information about the format of ground truth txt file and prediction txt file. 
For ground truth txt file, I guess it is <class><left><top><right><bottom>
For prediction txt file, is it <class><left><top><right><bottom><confidence> ?

**Comment by wangzhe0623** [comment]
@kkwong95 You can check 'sample' folder for reference

**Comment by kkwong95** [comment]
Yeah, I checked the 'sample' folder before I raise up this issue. 
I tried with your code with your sample txt file and It work. However, when I try to use mine, it has an error as below:

precision, recall, fppi, fppw, ap = self.computeAp(label)
ValueError: not enough values to unpack (expected 5, got 3)

My txt file's class name different from yours. My classes are 'car', 'motorbike', 'bus' and 'truck'. Is it the reason, my txt file cant run in your code ?

**Comment by wangzhe0623** [comment]
@kkwong95 Classes' name must be integer, like "0, 1, 2, 3", and it should be counted from "1".

**Comment by dreamhighchina** [comment]
是xyxy格式还是xywh格式？

**Comment by wangzhe0623** [comment]
@dreamhighchina xyxy，具体你看看代码吧。
